slurp/src/test/sum.js

```diff
@@ -0,0 +1,7 @@
+function sum(a, b) {
```

Comment:
Worth to have some comments here to explain this is a sample for using jest as you mentioned that in the RP description. This looks irrelevant to this project which might create confusion.